src/libraries/Common/src/Interop/OSX/Interop.libobjc.cs

```diff
@@ -63,7 +60,10 @@ internal static Version GetOperatingSystemVersion()
             return new Version(major, minor, patch);
         }
 
-        [DllImport(Libraries.libobjc, EntryPoint = MessageSendStructReturnEntryPoint)]
-        private static extern NSOperatingSystemVersion get_operatingSystemVersion(IntPtr basePtr, IntPtr selector);
+        [DllImport(Libraries.libobjc, EntryPoint = "obj_msgSend")]
+        private static extern NSOperatingSystemVersion osversion_objc_msgSend(IntPtr basePtr, IntPtr selector);
+
+        [DllImport(Libraries.libobjc, EntryPoint = "obj_msgSend_stret")]
+        private static extern void objc_msgSend_stret_rosversion(out NSOperatingSystemVersion osVersion, IntPtr basePtr, IntPtr selector);
```

Comment:
Nit: I think this should be `NSOperatingSystemVersion_objc_msgSend_stret` according to the Xamarin naming convention. 

The docs and examples are not 100% consistent, especially wrt casing. This is based on the dominant pattern from https://gist.github.com/rolfbjarne/981b778a99425a6e630c .